crates/tx-parser/src/filter_types.rs

```diff
@@ -0,0 +1,61 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use strata_primitives::{
+    buf::Buf32,
+    l1::BitcoinAddress,
+    params::{DepositTxParams, RollupParams},
+    sorted_vec::SortedVec,
+};
+
+use crate::utils::{generate_taproot_address, get_operator_wallet_pks};
+
+#[derive(Clone, Debug, BorshSerialize, BorshDeserialize)]
+pub struct TxFilterConfig {
+    /// For checkpoint update inscriptions.
+    pub rollup_name: String,
+
+    /// For addresses that we expect spends to.
```

Comment:
Who are we?

Reply:
[We are the champions](https://www.youtube.com/watch?v=04854XqcfCY)!! 😄  Removed the "we" references.